examples/knative/python/kn-py-vm-attr/function.yaml

```diff
@@ -0,0 +1,118 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: vcconfig-cm
+data:
+  VC_SERVER: vcsa.local # hostname of your vCenter
+  VC_USER: test@vsphere.local # service account name for the function
+  VC_SSLVERIFY: True # Verify the SSL certificate of the vCenter server?
+  VC_ATTR_OWNER: event-owner
+  VC_ATTR_CREATION_DATE: event-creation_date
+  VC_ATTR_LAST_POWEREDON: event-last_poweredon
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: vcconfig-secret
+  labels:
+    app: kn-vm-creation-attr-fn
+type: Opaque
+data:
+  VC_PASSWORD: Vk13YXJlMSEK # base64 encoded password
+---
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: kn-vm-creation-attr-fn-service
+  labels:
+    app: kn-vm-creation-attr-fn
+spec:
+  template:
+    metadata:
+      annotations:
+        autoscaling.knative.dev/maxScale: "1"
+        autoscaling.knative.dev/minScale: "1"
+    spec:
+      containers:
+      - image: lrivallain/kn-vm-creation-attr-fn
```

Comment:
Please use the official VMware project repo convention here (see other examples)